src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -156,7 +156,99 @@ mocha.describe('readings API', () => {
 
 				// Add C9 here
 
-				// Add C10 here
+				mocha.it('C10: 1 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and quantity units & kWh as BTU', async () => {
+					// Use predefined unit and conversion data
+					const unitData = unitDatakWh.concat([
+						//adding u1, u2, u3, u16
```

Comment:
u1 & u2 are defined in ``unitDataKWh`` and should not be redefined here. u3 & u16 should be concat here to define. The test case is failing on my machine and this is the reason why.